autohaven/autohaven_app/templates/signup.html

```diff
@@ -227,7 +227,11 @@ <h2>Your details</h2>
 
     <script>
```

Comment:
Instead of copying the confirmation modal into your template your template should extend base.html. You can take a look at the other pages that have the navbar at the top to get an idea on how to do it.